src/main/resources/modules/_bobsfirstmodule.json

```diff
@@ -0,0 +1,88 @@
+{
```

Comment:
The name of this module feels out of place. All of the other modules have a pseudo-condition name. Should this be part of the PR?

Reply:
@hankwallace for testing purpose now we have kept this module in code, but yes when we will raise pull on Synthea this won't be part of that pull request.

Reply:
Ok, understood.